tools/funclatency.py

```diff
@@ -62,6 +57,8 @@
     help="show a separate histogram per function")
 parser.add_argument("-r", "--regexp", action="store_true",
     help="use regular expressions. Default is \"*\" wildcards only.")
+parser.add_argument("-l", "--level", type=int,
+    help="set the level of nested or recursive functions")
```

Comment:
Looks great! Please change example file and man page as well since we added a new option here.

Reply:
Done.